src/rosdiscover/models/spawn_model.py

```diff
@@ -43,6 +43,11 @@ def spawn_model(c):
     with open('urdf.xml', 'w') as f:
         f.write(urdf_contents)
 
+    # #94 workaround to deal with inclusion of warning when xacro.py is used
```

Comment:
I think this assumes that there is only one <robot> block in each file. Is that always the case?

Reply:
Yep.